locale/pl/LC_MESSAGES/client.po

```diff
@@ -95,7 +95,7 @@ msgstr "Przykro nam o tym słyszeć. Jeśli nie ufasz już temu urządzeniu, to
 
 #: .es5/scripts/views/settings/clients.js:52
 msgid "Last active %(translatedTimeAgo)s"
-msgstr ""
+msgstr "Ostatnia aktywność: %(translatedTimeAgo)s"
```

Comment:
> yea this is the only way we can easily translate this at this time.

@vladikoff - why?

> Let me know if this does not work out in practice

As I understand correctly you just introduced external project dependency for Firefox localizations without even asking involved parties. Personally, I consider this a blocker. 

Maybe @flodolo could advise how to proceed.